webapp/maintenance/source/pages/index.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <transition name="fade" appear>
+    <ds-container width="medium">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-11T13:52:45Z" title="Wednesday, September 11th 2019, 3:52:45 pm +02:00">Sep 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,52 @@
+<template>
+  <transition name="fade" appear>
+    <ds-container width="small">
```


@mattwr18 didn't you say you changed this to `medium`?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-11T14:08:53Z" title="Wednesday, September 11th 2019, 4:08:53 pm +02:00">Sep 11, 2019</time>_

---


yes, I did!!